model/database.py

```diff
@@ -0,0 +1,36 @@
+from genericpath import exists
+from re import S
+from model.table import *
+
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+
+class Database:
+    def __init__(self, database_obj):
+        self.__validate(database_obj)
+        self.database_obj = database_obj
```

Comment:
who create an instance of Database in the future, should he/she know if it has a database_obj or not?